src/test/java/org/jenkinsci/plugins/pipeline/utility/steps/conf/ReadYamlStepSystemProperties.java

```diff
@@ -0,0 +1,28 @@
+package org.jenkinsci.plugins.pipeline.utility.steps.conf;
+
+import org.junit.Rule;
+import org.junit.Test;
+import org.jvnet.hudson.test.FlagRule;
+import org.jvnet.hudson.test.JenkinsRule;
+
+import static org.junit.Assert.assertEquals;
+
+public class ReadYamlStepSystemProperties {
```

Comment:
All test classes/suite names needs to end with `Test` otherwise Maven surefire will skip them.

```suggestion
public class ReadYamlStepSystemPropertiesTest {
```